File: array-2574.py
nums = [10,4,8,3]

leftSum = []
rightSum = []

leftSum.append(0)

l = 0
for i in range(0, len(nums)-1):
    l += nums[i]
    leftSum.append(l) 

r = 0
for i in range(len(nums)-1, 0, -1):
    r += nums[i]
    rightSum.append(r)

rightSum.reverse()
rightSum.append(0)

ans = []
for i in range(0, len(nums)):
    ans.append(abs(leftSum[i] - rightSum[i]))

print(ans)

File: array2574.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;
int main() {

    int n,;
    cin>>n;
    vector<int>nums(n);
    for(int i=0;i<n;i++) {
        cin>>nums[i];
    }
    vector<int>leftsum(n);
    vector<int>rightsum(n);
    vector<int>result(n);
    for(int i=0;i<n;i++) {
        int s=0;
        for(int j=0;j<i;j++) {
            s+=nums[j];
        }
        leftsum[i]=s;
    }
        
}
    for(int i=0;i<n;i++) {
        int s=0;
        for(int j=i+1;j<n;j++) {
            s+=nums[j];
        }
            rightsum[i]=s;
    }
        
    for(int i=0;i<n;i++) {
         result[i]=abs(leftsum[i]-rightsum[i]);

    }
    for(int c: result) {
        cout<<c;
    }

    

    return 0;
}